crates/fluvio-smartengine/src/engine/wasmedge/imp.rs

```diff
@@ -0,0 +1,114 @@
+use super::instance::{WasmEdgeContext, WasmEdgeFn, WasmEdgeInstance};
+
+use tracing::debug;
```

Comment:
can you reorder terms in of: 
- std
- third party
- fluvio dep

Reply:
That's OK. Are you going to review the PR now? I will resolve such issues if general ideas LGTY.

P.S. this PR is generally complete except minor points.

Reply:
Can you split this into PR just related to without WasmEdge?  We need to discuss how to maintain WasmEdge related code.

Thanks

Reply:
That's OK. But I'd like to elaborate a bit: only folder `wasmedge` is WasmEdge-specific code, and that's ~300 lines of simple code. To discuss how to maintain WasmEdge related code, and to get the ideas of how the common code work, it might be better to keep WasmEdge related code now. This can actually make it easier to review this PR.

I can remove the WasmEdge related code after the code review if that's not decided.

Reply:
I would like have review by team but can't seem to add reviewers

Reply:
I created a PR to the main repo https://github.com/infinyon/fluvio/pull/3257